modules/angular-meteor-collection.js

```diff
@@ -53,14 +54,61 @@ angularMeteorCollection.factory('AngularMeteorCollection', [
       return allPromise;
     };
 
-    AngularMeteorCollection.remove = function (keys) {
-      // remove collection
-      if (!keys) _.pluck(this, '_id');
-      // remove single doc
-      keys = [].concat(keys);
+    AngularMeteorCollection._upsertDoc = function(doc, useUnsetModifier) { 
```

Comment:
@DAB0mB Hey, what kind of collection is it supposed to be here 
https://github.com/barbatus/angular-meteor/blob/master/modules/angular-meteor-object.js#L21?
Mongo.Collection doesn't have upsertObejct method at all.